package.json

```diff
@@ -14,8 +14,8 @@
     "start": "npm run storybook",
     "storybook": "start-storybook -p 6006 -s src/static",
     "build-storybook": "build-storybook -s ./src/static",
-    "test": "npm run test-lint && prettier --check './src/**/*' && node ./scripts/build.test.js && npm run test-ui",
-    "test-lint": "eslint src/",
```

Comment:
That was never recursive to all folders. We now line and prettify almost everything for more consistency.